Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -279,17 +280,22 @@ public void selectClick() {
     protected static final String INCOMPLETE_DATA = "errorDataIncomplete";
 
     /**
-     * Prepare.
+     * Prepare template and project for which new process will be created.
      *
-     * @return empty String
+     * @param projectId
```

Comment:
You should switch the ordering in JavaDoc to match ordering of method parameters.